src/init.cpp

```diff
@@ -31,7 +31,9 @@
 #include "wallet/wallet.h"
 #include "wallet/walletdb.h"
 #endif
-
+#if ENABLE_ZMQ
```

Comment:
Nit: IMHO the ZMQ header stuff should go below std headers and below boost, openssl etc (header group ordering).